ompy/matrix.py

```diff
@@ -99,24 +101,29 @@ def __init__(self,
             Ex: The excitation energies using midbinning.
             std: The standard deviations at each bin of `values`
             path: Load a Matrix from a given path
-            shape: Tuple (len(Ex), len(Ex)) to create a matrix with 0 counts.
             state: An enum to keep track of what has been done to the matrix.
                 Can also be a str. like in ["raw", "unfolded", ...]
+            shape: Depreciated. Use `ZerosMatrix` instead.
 
         """
+        if shape is not None:
+            warnings.warn("Creating a Matrix with zeros as entries by the "
+                          "shape argument is depreciated. Use ZerosMatrix "
+                          "instead.", DeprecationWarning)
+            values = ZerosMatrix(shape=shape, Ex=Ex, Eg=Eg).values
```

Comment:
this is slightly ugly, but would preserve the current functionality  for some time (until one cleans up)

Reply:
I think it looks alright, gets the job done. Isn't meant to last forever anyways:)